src/web/routes/application/do-you-live-in-scotland/validate.js

```diff
@@ -0,0 +1,11 @@
+const { check } = require('express-validator/check')
+const { translateValidationMessage } = require('../common/translate-validation-message')
+const { YES, NO } = require('../common/constants')
+
+const validate = [
+  check('doYouLiveInScotland').isIn([YES, NO]).withMessage(translateValidationMessage('validation:selectYesOrNo'))
```

Comment:
Could this be moved into somewhere common? I imagine it's the same of are you pregnant?

Reply:
This is a fair point but I would say that it is unnecessary abstraction, adding more problems (in terms of indirection during debugging) than it solves - if most validation have inline messages but this abstraction breaks that rule, it makes mentally parsing the code that bit more difficult.

It would require a function that takes in the field which would in turn require adding to the validation array:
```
const checkIsYesOrNo = (fieldName) => ...

const validate = [
  ...checkIsYesOrNo('doYouLiveInScotland'),
  check('SomethingElse').withMessage('message is right here'),
  check('theNextThing').withMessage('message is right here')
]
```
I don't think it's a massive overhead to have a small amount of duplication in this code.

Reply:
Having said that, its not a difficult change so happy to make the update if others don't agree with me.

Reply:
... on second thoughts I have addressed this with a refactor!!